### PR TITLE
BaSP-TG-110: Employee Profile

### PR DESCRIPTION
**Pull request by GuidoCerioni**
Added Profile component. It allows the user to see his profile and update his data.
I added some Redux logic to Header component so that we have some data in the state when we call Profile. This will change in the future.

### CODE REVIEW

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]

**Review by Gearyandres94 — APPROVED** [review]
Great job!!

File: src/components/Shared/Profile/index.js [hunk 1]
```diff
@@ -121,9 +125,16 @@ const Profile = () => {
...
+    setChangePassword(true);
+    setValue('password', '');
+    setValue('rpassword', '');
+    setValue('oldPassword', '');
+  };
+
   const onSubmit = (data) => {
     if (data.oldPassword === employee.password) {
-      console.log(data);
+      console.log('asdasd');
```
Comment (Gearyandres94) [inline src/components/Shared/Profile/index.js:137]: delete this console log

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
Great job! You just need to delete a line of code.

**Review by Gearyandres94 — APPROVED** [review]
Great job!!!

**Review by HernanCazeneuve — DISMISSED** [review]

**Review by Gearyandres94 — DISMISSED** [review]
Good job!!

**Review by HernanCazeneuve — APPROVED** [review]
great job

**Review by Gearyandres94 — APPROVED** [review]
Good job!

**Review by HernanCazeneuve — APPROVED** [review]

**Review by CarVigil — APPROVED** [review]

**Pull request merged**